code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -155,7 +159,13 @@
 
 	for(var/organ in has_organ)
 		var/organ_type = has_organ[organ]
+		//world << "[organ_type]"//debug print
 		H.internal_organs_by_name[organ] = new organ_type(H,1)
+		H.internal_organs += new organ_type
```

Comment:
Oh and finally it looks like you're making 2 separate organs here for each organ tag hence the weird duplication goofery.